### ISSUE TITLE
Output ISO8601 dates

### ISSUE THREAD

**Issue by 4e554c4c** [mfreeborn/heliocron#44]
Currently heliocron reports events as `Never` or very similar to `date '+%Y-%m-%d %H:%M:%S %:z'`, this is nonstandard and it would be very easy to output in ISO8601 instead, as `chrono::DateTime` has a [`to_rfc3339`](https://docs.rs/chrono/0.4.19/chrono/struct.DateTime.html#method.to_rfc3339) method. Furthermore, there is no way to control the timezone of output dates.

**Comment by mfreeborn** [comment]
Hmm,  I see you are parsing the output of the `report` command in your shell script. What I'm considering is whether the output of `report` is for humans to read or programs to read.

At the moment, it is designed for humans to read, hence the use of nonstandard date formatting, because it is easier to read `2022-06-06 13:19:03 +01:00` than `2022-06-06T13:19:03+01:00`.

I wonder if what we actually need is an optional format flag for the output of the report. Something like `heliocron report --json`, which would then output machine-parseable Json e.g `{"date": "2022-06-06T13:19:03+01:00", "sunrise": "2022-06-06T04:31:27+01:00", ...etc}`.

Proposed schema (opinions heartily welcomed):
```
{
  "location": {"latitude": float, "longitude": float}
  "date": string,
  "day_length": integer,
  "solar_noon": string,
  "sunrise": string,
  "sunset": string,
  "dawn": {"civil": string, "nautical": string, "astronomical": string}, // could un-nest the dawn and dusk fields if we want...
  "dusk": {"civil": string, "nautical": string, "astronomical": string},

}
```
Where `latitude`/`longitude` are coordinates in decimal degree notation, `day_length` is an integer number of seconds, and any datetime is an rfc3339-formatted string. An event which occurs `Never`... could be an empty string?

**Comment by 4e554c4c** [comment]
Yeah! this sounds perfect

I was thinking about a "machine output" option, and this would work great

**Comment by mfreeborn** [comment]
@4e554c4c  Thanks very much for releasing `tokio-walltime` v0.1.2 - the newly released v0.6.1 of `heliocron` now does what you need :)
